404.php

```diff
@@ -2,30 +2,44 @@
 get_header();
 ?>
 
-<section id="main-content">
-  <div id="page-404" class="grid-row align-items-center">
-    <div class="grid-item item-s-12 text-align-center font-size-large">
+<main id="main-content">
+  <section id="404" class="margin-top-extra margin-bottom-large">
+    <div class="container">
+      <div class="grid-row align-items-center">
+        <div class="grid-item item-s-12 text-align-center font-size-large">
 <?php
 switch(qtranxf_getLanguage()) {
   case 'es':
 ?>
-      <span>¡404 No se encontraron los <span id="globie-part">Pies</span>!</span>
+          <span>¡404 No se encontraron los <span id="globie-part">Pies</span>!</span>
```

Comment:
Also I think there should be a colon after the 404 otherwise it could be 404 pairs of feet not found?!